.github/workflows/tag_image_push_rhel.yml

```diff
@@ -121,6 +132,27 @@ jobs:
         run: |
           docker login ${SCAN_REGISTRY} -u ${SCAN_REGISTRY_USER} -p ${SCAN_REGISTRY_PASSWORD}
 
+      - name: Check if LTS release
```

Comment:
this is duplicate code
anyway can be be moved to shared (action)?

Reply:
Done: https://github.com/hazelcast/hazelcast-docker/pull/808/commits/9f767d6080d2c0ec6e9488cc87da69ae399fe652